tests/integration/client/feedback/dataset/remote/test_dataset.py

```diff
@@ -17,7 +17,14 @@
 from uuid import UUID
 
 import pytest
-from argilla import FeedbackRecord
+from argilla import (
+    FeedbackRecord,
+    FloatMetadataProperty,
+    IntegerMetadataProperty,
+    TermsMetadataProperty,
+    TextField,
+    TextQuestion,
+)
```

Comment:
Do we really want to import these `from argilla` instead of from their respective module? e.g.

```suggestion
from argilla.client.feedback.schemas.records import FeedbackRecord
from argilla.client.feedback.schemas.metadata import FloatMetadataProperty, IntegerMetadataProperty, TermsMetadataProperty
from argilla.client.feedback.schemas.fields import TextField
from argilla.client.feedback.schemas.questions import TextQuestion
```

Reply:
Do you mean in tests? I would like to use them as a user should use them, so yes, they should be imported from argilla. Otherwise, we cannot check that expected schemas have been properly configured in the main module

Reply:
Ok cool, then we should probably create a separate PR at some point to refactor all the integration tests to use `import argilla as rg` instead, as that's what we're pointing users in the docs